hare.spec

```diff
@@ -56,7 +53,7 @@ Requires: puppet-agent >= 6.13.0
 Requires: facter >= 3.14.2
 %endif
 Requires: jq
-Requires: cortx-motr = %{h_motr_version}
+Requires: cortx-motr
```

Comment:
So it means we can use any motr rpm versions with latest hare rpm.
How we will make sure whether motr rpm installed is compatible with latest Hare rpm?

Reply:
I have the same question that @SwapnilGaonkar7 does - this change removes all version checks between hare and motr and reduces it to checking whether motr is installed. Is that a safe assumption? Would it not break anything else?

Reply:
So with this change we can now build Hare with a different motr version and a different motr version can be installed. I understand this is okay for small changes in-order to save some build time. But If there are changes related to protocol or data structures for example, they could be different in Hare than underlying motr. Such cases, would further lead to failure. The version check was used to safe guard from the latter. Same is true for RGW. With disabling, now  it is possible that the we might hit some crash at some later point in time during execution.

Reply:
@rkothiya @madhavemuri any comments on the concerns that @mssawant mentioned here?

Reply:
@mssawant : At ha interface level this check is already removed. If any API level changes happens we will update both hare and motr in the image.  This is not only for build purpose, we are removing dependency from both Hax and S3, assumption is that API's of libmotr will change rarely going forward and if that happens so which can be managed at release of CORTX image level.